src/main/resources/db/migration/courtcase/V2129__add_personid_to_defendant.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+    CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
I don't believe this should be necessary as this extension should have been added in a previous migration.. it won't do any harm given the `IF NOT EXISTS` but I'm guessing you added it because you were having issues with it otherwise? 🤔 Are you aware of [this known issue](https://github.com/ministryofjustice/court-case-service#known-issues)? Not sure if that might be relevant?